packages/app/src/Pages/settings/Relays.tsx

```diff
@@ -31,6 +31,10 @@ const RelaySettingsPage = () => {
     }
   }
 
+  function exportRelays() {
+    navigator.clipboard.writeText(Object.keys(relays).join("\r\n"));
```

Comment:
Maybe we can join them by one of the separators in the split below, why `\r\n`

Reply:
I want the list to be splited by new lines, If I need to paste the it on x IM clients